flask_user/forms.py

```diff
@@ -210,7 +221,7 @@ def validate(self):
             user, user_email = user_manager.find_user_by_email(self.email.data)
 
         # Handle successful authentication
-        if user and user.password and user_manager.verify_password(self.password.data, user):
+        if user and user_manager.verify_password(self.password.data, user):
```

Comment:
`self.password.data` is on the form itself? I don't know how that could fail unless you're overriding the `password` input on the form, in which case you can override the `validate` method as well.